### PR TITLE
new blog post about joining hanami core team

### PR DESCRIPTION
**Pull request by marionschleifer**

### CODE REVIEW

**Comment by AlfonsoUceda** [comment]
@marionschleifer nice story and post! :D

**Comment by marionschleifer** [comment]
@AlfonsoUceda thanks a lot :)

**Comment by davydovanton** [comment]
hey, post looks great. @AlfonsoUceda @cllns @jodosha WDYT can we merge this?

**Comment by AlfonsoUceda** [comment]
I think @marionschleifer can merge this.

**Comment by marionschleifer** [comment]
@davydovanton @AlfonsoUceda Thank you :) @jodosha said it would be better to wait until after the stable release of Hanami 1.0. There, he will introduce the new core team. Otherwise, nobody even knows who I am ;-)

**Comment by graywolf** [comment]
hm... should I read it here or wait till it's published... tough question :)

**Comment by davydovanton** [comment]
@marionschleifer hey, any updates? :)

**Comment by marionschleifer** [comment]
@jodosha Can we merge this now?

File: source/blog/2017-01-22-meeting-luca-by-marion.html.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: How I came to be in the Hanami Core Team
+date: 2017-01-22 17:00 UTC
```
Comment (cllns) [inline source/blog/2017-01-22-meeting-luca-by-marion.html.markdown]: Can you update this when it's merged? :)

Comment (marionschleifer) [inline source/blog/2017-01-22-meeting-luca-by-marion.html.markdown]: @cllns Yes, of course 😃

**Comment by AlfonsoUceda** [comment]
Is this ready to be deployed?

**Comment by marionschleifer** [comment]
@AlfonsoUceda I think maybe @jodosha should have a look at it if it's ok now 😄

**Comment by jodosha** [comment]
Closing as stale contribution. @marionschleifer Feel free to resume the work.

**Pull request closed without merging**